lib/trino-parquet/src/main/java/io/trino/parquet/predicate/PredicateUtils.java

```diff
@@ -181,6 +185,18 @@ private static Map<ColumnDescriptor, Statistics<?>> getStatistics(BlockMetaData
         return statistics.buildOrThrow();
     }
 
+    private static Map<ColumnDescriptor, Long> getColumnValueCounts(BlockMetaData blockMetadata, Map<List<String>, ColumnDescriptor> descriptorsByPath)
+    {
+        ImmutableMap.Builder<ColumnDescriptor, Long> columnValueCounts = ImmutableMap.builder();
+        for (ColumnChunkMetaData columnMetaData : blockMetadata.getColumns()) {
+            ColumnDescriptor descriptor = descriptorsByPath.get(Arrays.asList(columnMetaData.getPath().toArray()));
```

Comment:
nit: `Arrays.asList(X.toArray())` seems redundant.

Reply:
That's needed to convert `ColumnPath -> String[] -> List<String>`
I'm not sure we didn't keep things simpler by just sticking to `ColumnPath` or `String[]` everywhere, but that's an unrelated change and I need to follow existing convention here.